wbooks-api/spec/factories/books.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :book do
+    genre     { Faker::Book.genre }
+    author    { Faker::Book.author }
+    image     { Faker::Internet.url }
+    title     { Faker::Book.title }
+    publisher { Faker::Book.publisher }
+    year      { '2018' }
```

Comment:
Two things about this

- If you need to use a constant value for an attribute in a factory you can just remove the { and }
  ```
  year '2018'
  ```
  because the { and } are for a block (that is, every time you create an object using the factory it'll execute the block and get a new result)

- While in this case it's not vital, you can use date to generate years. Use the date faker: https://github.com/stympy/faker/blob/master/doc/date.md
and then call .year for the result